recipes-support/rauc-hawkbit-updater/rauc-hawkbit-updater_%.bbappend

```diff
@@ -0,0 +1,14 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+do_install_append () {
+    rm ${D}${sysconfdir}/${PN}/config.conf
+    install -d ${D}/mnt/config/hawkbit
+    install -m 644 ${S}/config.conf.example ${D}/mnt/config/hawkbit/config.cfg
+    ln -s /mnt/config/hawkbit/config.cfg ${D}/${sysconfdir}/rauc-hawkbit-updater/config.conf
```

Comment:
Remove the `/` in `${D}/${sysconfdir}`. `${sysconfdir}` evaluates to something like `/etc` where the `/` is already included and thus will form a valid path in conjunction with `${D}`.